### DISCUSSION TITLE
custom Get is 404

### DISCUSSION THREAD

**Discussion by langziyang** [api-platform/core#6378] [Q&A]
![image](https://github.com/api-platform/core/assets/2345387/056d5fc5-32f7-4a9b-a9b9-e4259a23db68)

when i call /api/surveys/export  i get 404 error.

but if i change /surveys/export to /surveys/data/export then it working fine. please tell me how make /surveys/export it working

**Comment by soyuka** [comment]
put the route first, or else it'll match the {id}
